Shared/AuthenticationKeychainInfo.swift

```diff
@@ -63,6 +67,8 @@ open class AuthenticationKeychainInfo: NSObject, NSCoding {
         self.useTouchID = aDecoder.decodeAsBool(forKey: "useTouchID")
         if aDecoder.containsValue(forKey: "isPasscodeRequiredImmediately") {
             self.isPasscodeRequiredImmediately = aDecoder.decodeAsBool(forKey: "isPasscodeRequiredImmediately")
+        } else if let interval = aDecoder.decodeObject(forKey: "requiredPasscodeInterval") as? NSNumber {
```

Comment:
Is this a key from 1.6 that is no longer used in 1.7? (besides here)?

Reply:
Yes, this is used for the isPasscodeRequiredImmediately bool, It will fall in this case only once.